tests/src/openshift-local-extension.spec.ts

```diff
@@ -109,22 +110,51 @@ test.describe.serial('Red Hat OpenShift Local extension verification', () => {
   });
 
   test.describe.serial('Red Hat OpenShift Local extension handling', () => {
-    test('Extension can be disabled', async ({ navigationBar }) => {
+    test('Extension can be disabled', async ({ page, navigationBar }) => {
       const extensions = await navigationBar.openExtensions();
       playExpect(await extensions.extensionIsInstalled(extensionLabelCrc)).toBeTruthy();
       const extensionCard = await extensions.getInstalledExtension(extensionLabelNameCrc, extensionLabelCrc);
       await playExpect(extensionCard.status).toHaveText(activeExtensionStatus);
       await extensionCard.disableExtension();
       await playExpect(extensionCard.status).toHaveText(disabledExtensionStatus);
+      //checking dashboard assets
+      const dashboard = await navigationBar.openDashboard();
+      await playExpect(dashboard.openshiftLocalProvider).toHaveCount(0);
+      //checking settings/resources assets
+      await navigationBar.openSettings();
+      const settingsBar = new SettingsBar(page);
+      const resourcesPage = await settingsBar.openTabPage(ResourcesPage);
+      const openShiftLocalCard = resourcesPage.featuredProviderResources.getByRole('region', {name: 'crc'});
+      await playExpect(openShiftLocalCard).toHaveCount(0);
+      //checking settings/preferences assets
+      const preferencesPage = await settingsBar.openTabPage(PreferencesPage);
+      const preferencesTab = await preferencesPage.getTab();
+      playExpect(await preferencesTab.getByLabel('Extension: Red Hat OpenShift Local').count() == 0);
```

Comment:
You are reusing this locator across the file, you can at least define the locator once, ie. as constant. This can apply for more locators. You can have one definition of the locator and then check for it in excepts.